infrastructure/data_access/http/synthetics_repo.py

```diff
@@ -77,6 +78,20 @@ def transform_to_internal_mesh_columns(input_columns: List[V202101beta1MeshColum
                 health=input_column.metrics.packet_loss.health,
                 value=int(input_column.metrics.packet_loss.value),
             ),
+            health=transform_to_internal_health_items(input_column.health),
         )
         columns.append(column)
     return columns
+
+
+def transform_to_internal_health_items(input_health: List[V202101beta1MeshMetrics]) -> List[HealthItem]:
+    health: List[HealthItem] = []
+    for h in input_health:
+        item = HealthItem(
+            jitter_microsec=int(h.jitter.value),
```

Comment:
Let's do the scaling to milliseconds here as opposed to at every place we use the data. The value though should be `float`. We may end=up showing bunch of zeroes for jitter smaller than 1 ms.

```
jitter_millisec = h.jitter.value / 1000
...
```

Reply:
Updating to milliseconds